packages/trackerless-network/protos/NetworkRpc.proto

```diff
@@ -47,48 +43,44 @@ enum StreamMessageType {
   GROUP_KEY_RESPONSE = 2;
 }
 
-enum EncryptionType {
-  NONE = 0;
-  RSA = 1;
-  AES = 2;
+enum ContentType {
+  JSON = 0;
+  BINARY = 1;
 }
 
-message EncryptedGroupKey {
-  string groupKeyId = 1;
-  string encryptedGroupKeyHex = 2;
-  optional string serialized = 3;
+enum EncryptionType {
+  NONE = 0;
+  AES = 1;
 }
 
 message StreamMessage {
   StreamMessageType messageType = 1;
-  optional EncryptionType encryptionType = 2;
-  bytes content = 3;
-  string signature = 4;
-  MessageRef messageRef = 5;
-  optional MessageRef previousMessageRef = 6;
-  optional string groupKeyId = 7;
-  optional EncryptedGroupKey newGroupKey = 8;
+  ContentType contentType = 2;
+  EncryptionType encryptionType = 3;
+  bytes content = 4;
+  bytes signature = 5;
+  MessageRef messageRef = 6;
```

Comment:
Keeping to existing terminology I would rename the type to `MessageID` and field name to `messageId` unless there is a specific reason to deviate from this naming convention.